packages/tour-kit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased Changes
+
+- Added the spotlight interactivity feature, which is disabled by default so this is not a breaking change
```

Comment:
I think we don't need to add this changelog since we haven't released 1.0.0. Could you remove it?

Reply:
Ok done!